scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/HsqldbServer.java

```diff
@@ -125,21 +125,25 @@ public void addCatalog(Path defaultLocation, Path hibernateConfiguration) throws
 
         String connectionUrl = hibernateProperties.getProperty(PROP_HIBERNATE_CONNECTION_URL);
 
-        String catalogLocation = identifyCatalogLocationFromConnectionUrl(connectionUrl);
-        String catalogName = identifyCatalogNameFromConnectionUrl(connectionUrl);
-        String catalogPassword = hibernateProperties.getProperty(PROP_HIBERNATE_CONNECTION_PASSWORD);
-        String catalogUsername = hibernateProperties.getProperty(PROP_HIBERNATE_CONNECTION_USERNAME);
-
-        Path catalogPath;
-        if (catalogLocation == null) {
-            // Catalog path references a file.
-            // Double resolve is used to get a dedicated folder with the catalog name.
-            catalogPath = defaultLocation.resolve(catalogName).resolve(catalogName);
-        } else {
-            catalogPath = Paths.get(catalogLocation);
+        if (connectionUrl.toLowerCase().contains("hsqldb")) {
```

Comment:
So, this would prevent error messages in the logs when using another DB than HSQL, correct?

Reply:
we hope so